### PR TITLE
DDO-790 rawls k8s initial

### PR DESCRIPTION
**Pull request by mflinn-broad**
Initial pr for the rawls helm chart

### CODE REVIEW

File: charts/rawls/templates/_deployment.tpl [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            fieldRef:
+              fieldPath: metadata.name
+        - name: PROMETHEUS_ARGS
+          value: "-javaagent:/etc/prometheusjmx/prometheusjmx.jar=9090:/etc/rawls-cm/prometheusJmx.yaml"
+        command: ["/bin/bash"]
+        args:
+        - '-c'
+        - >-
+          java ${JAVA_OPTS}
+          {{- if .Values.prometheus.enabled }}
```
Comment (gmalkov) [inline charts/rawls/templates/_deployment.tpl:81]: IMO if we are going to have this flag we should make the chart work without Prometheus if it is disabled. See [this commit](https://github.com/broadinstitute/terra-helm/pull/182/commits/1f02162a633c959fe7c484183254f74045dc11b2) for an example of the changes needed.

File: charts/rawls/values.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  # deploymentDefaults.serviceName -- What to call the Service
+  serviceName: null
+  # deploymentDefaults.serviceIP -- Static IP to use for the Service. If set, service will be of type LoadBalancer
+  serviceIP: null
+  # deploymentDefaults.legacyResourcePrefix -- What prefix to use to refer to secrets rendered from firecloud-develop
+  # @default deploymentDefaults.name
+  legacyResourcePrefix: null
+
+# A map of rawls deployments. In Terra's production environment,
+# rawls is deployed as 2 separate services (frontend and backend) with different configurations.
```
Comment (gmalkov) [inline charts/rawls/values.yaml:69]: This isn't super clear. Is a map supposed to go here? Maybe give an example?

**Review by gmalkov — APPROVED** [review]

File: charts/rawls/values.yaml (hunk 2 above)
Comment (mflinn-broad) [reply charts/rawls/values.yaml:69]: ah I just copy pasted this from the cromwell chart and did a find and replace for rawls... but yeah good point it is a little unclear.

File: charts/rawls/templates/_deployment.tpl (hunk 1 above)
Comment (mflinn-broad) [reply charts/rawls/templates/_deployment.tpl:81]: I'm not sure this flag is necessary. I can't think of a situation where we'd want to set it to false off the top of my head. I actually went through and removed the flag, but I missed this one. 

I think if we ever do have a use case for this flag its easy enough to go in and add it.

File: charts/rawls/values.yaml (hunk 2 above)
Comment (mflinn-broad) [reply charts/rawls/values.yaml:69]: ah I remember this now, this was a section that was needed in the cromwell chart but not in rawls, forgot to delete the comment

**Pull request merged**